sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs

```diff
@@ -309,7 +309,7 @@ private async Task ExtendResourceGroupExpirationAsync()
             string clientSecret = GetOptionalVariable("CLIENT_SECRET");
             string authorityHost = GetOptionalVariable("AZURE_AUTHORITY_HOST");
 
-            if (tenantId == null || clientId == null || clientSecret == null || authorityHost == null)
+            if (tenantId == null || clientId == null || clientSecret == null || authorityHost == null || ResourceManagerUrl == null)
```

Comment:
I noticed this was missing while testing - if there is no ResourceManagerUrl env var we cannot extend RG expiration.